livy/src/main/resources/interpreter-setting.json

```diff
@@ -118,6 +118,11 @@
         "defaultValue": "1000",
         "description": "Max number of Spark SQL result to display."
       },
+      "zeppelin.livy.spark.sql.truncate": {
+        "propertyName": "zeppelin.livy.spark.sql.truncate",
+        "defaultValue": "true",
+        "description": "If true, truncate strings greater than 20 characters."
```

Comment:
oh got it, sorry, so there's one for max line number.
perhaps to rename this to clarify? `zeppelin.livy.spark.sql.truncate.string.value` or something?